### DISCUSSION TITLE
Voltage mod means I cannot calibrate properly

### DISCUSSION THREAD

**Discussion by BAREBONESLDN** [eh2k/squares-and-circles#116] [Q&A]
Hi,

I did the voltage mod which works fine but means I cannot complete the final stage of the calibration as it's now out of range.
Should I calibrate first and  then add the extra resistor for the voltage mod?

Thanks!

**Comment by eh2k** [comment]
Hi, you should definitely calibrate after the mod. If you turn the encoder while holding it down, you can first roughly adjust the offset. For fine adjustment, turn the encoder without pressing it.
